zerver/views/pointer.py

```diff
@@ -52,3 +52,12 @@ def get_profile_backend(request, user_profile):
         result['max_message_id'] = messages[0].id
 
     return json_success(result)
+
+def deactivate_user_backend(request, user_profile):
+    # type: (HttpRequest, UserProfile) -> HttpResponse
+    return _deactivate_user_profile_backend(request, user_profile)
+
+def _deactivate_user_profile_backend(request, target):
+    # type: (HttpRequest, UserProfile) -> HttpResponse
+    do_deactivate_user(target)
+    return json_success({})
```

Comment:
Okay I consolidated the two functions! I actually started off trying to use the admin function in  `users.py`, but I ended up having to change the function slightly by removing the email parameter and deleted a lot of unnecessary code involving bots. How would I check if a user is the last admin? Is there a list of active admins stored?